src/systemctl/systemctl.h

```diff
@@ -44,6 +44,8 @@ enum dependency {
         _DEPENDENCY_MAX
 };
 
+#define DEFAULT_ARG_JOB_MODE "replace"
```

Comment:
i'd really prefer if we'd solve this in a more abstract, encapsulating way. instead of adding a macro for this and encoding the ternary op everywhere, just do it like this:

```c
static inline const char *arg_job_mode_get(void) {
        return arg_job_mode ?: "replace";
}
```

and then use that function wherever we need it. 

My thinking here: better abstract the algorithm instead of exporting the parameters for it.

Reply:
Yep, that's better. I renamed the param to `_arg_job_mode` though, and called the function `arg_job_mode()`. I think this reduces the chance of someone passing the function pointer instead of the variable by mistake.